build.cake

```diff
@@ -125,6 +125,7 @@ Task("Restore-NuGet-Packages")
             "https://dotnet.myget.org/F/dotnet-core/api/v3/index.json",
             "https://dotnet.myget.org/F/cli-deps/api/v3/index.json",
             "https://api.nuget.org/v3/index.json",
+            "C:/Users/mb/src/gh/nuget.client.netstandard/artifacts/NuGet.PackageManagement/15.0/bin/Release/"
```

Comment:
Remove once https://github.com/aelij/NuGet.Client/pull/1 is merged and new package is pushed to NuGet